dags/cache_favicons.py

```diff
@@ -0,0 +1,103 @@
+import asyncio
```

Comment:
I wasn't sure know how this would play out in Dagster. 

Should be fine as long as we just define the coroutine in the single asset: https://github.com/dagster-io/dagster/discussions/16269